data/json/items/tool_armor.json

```diff
@@ -2211,6 +2211,51 @@
     },
     {
         "type" : "TOOL_ARMOR",
+        "id" : "bagpipes",
+        "name" : "bagpipes",
+        "name_plural": "bagpipes",
+        "category" : "tools",
+        "weight" : 2300,
+        "color" : "green",
+        "covers" : ["TORSO"],
```

Comment:
I figure since it's not necessary to blow into them constantly, and they were made to wear and use during battle, they ought to be fairly comfortable to wear.